x-pack/filebeat/module/azure/auditlogs/_meta/fields.yml

```diff
@@ -52,16 +56,17 @@
           type: keyword
           description: >
             ID
-        - name: activityDateTime
+        - name: activity_datetime
           type: keyword
```

Comment:
hmm do we have a date type for timestamps?

Reply:
"date" , I have made the changes